### DISCUSSION TITLE
Block some sites and delay others

### DISCUSSION THREAD

**Discussion by ph0uad** [proginosko/LeechBlockNG#475] [Q&A] [answered]
I want to have *.edu allowed, gmail.com delayed, and everything else blocked.

How?

**Comment by proginosko** [comment, marked as answer]
You need to use two block sets.

First set, using blocking page:
`*`
`+*.edu`
`+gmail.com`
`+mail.google.com`

Second set, using delaying page:
`mail.google.com`

**Comment by ph0uad** [comment]
Very simple. Thank you.
